addons/auth_totp_mail_enforce/views/res_config_settings_views.xml

```diff
@@ -8,23 +8,13 @@
             <field name="priority" eval="40"/>
             <field name="inherit_id" ref="base_setup.res_config_settings_view_form"/>
             <field name="arch" type="xml">
-                <xpath expr="//div[@id='allow_import']" position="before">
-                    <div class="col-12 col-lg-6 o_setting_box" id="auth_totp_policy">
-                        <div class="o_setting_left_pane">
+                <xpath expr="//setting[@id='allow_import']" position="before">
```

Comment:
Maybe a good opportunity to uniformize the xpaths of all the views ?
```suggestion
                <setting id="allow_import" position="before">
```

Reply:
Hello @Feyensv ,

Thank you for your comment.
Indeed, the second step will be to think about the possibility to create a specific setting view and not only a jsClass, for the reasons you enumerate and others.

About the uniformization of the xpath, it's out of scope of this PR. And maybe it would be better if it's done across all the arches and not only for the setting's one (so in another task).